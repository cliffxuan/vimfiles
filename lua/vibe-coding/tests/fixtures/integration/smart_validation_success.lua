return {
  name = 'Smart Validation - Successfully Fixed Context',
  description = 'Test that smart validation correctly fixes context lines that dont match original file',
  should_succeed = true,
  tags = { 'integration', 'smart_validation', 'context_fix' },
  
  file_path = 'example.py',
}