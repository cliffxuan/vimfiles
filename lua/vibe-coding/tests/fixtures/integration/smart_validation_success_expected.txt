def calculate_total(items):
    """Calculate the total price of items."""
    total = 0
    for item in items:
        total += item.price * item.quantity
    return total

def process_order(order):
    items = order.get_items()
    return calculate_total(items)